### ISSUE TITLE
dynamic sorting by usage frequency

### ISSUE THREAD

**Issue by itc-ger** [helloworld1/FreeOTPPlus#120]
The popularity of TOTP-2FA (a.k.a GoogleAuthenticator) is growing
and my personal token list in FreeOTP+ is growing the same way.

I have found that I use part of the token very frequently while others only in rare occasions.
Sorting a lot of tokens manually is time consuming and annoying too. Especially if it's not clear if a token is often used in the future or not (therefore one have to reorder the tokens repeatedly).

So my idea is to have the token list sorted by the frequency of their use. Not to be misunterstood, this should be optional!
Can you consider to implement such a feature, please?

**Comment by helloworld1** [comment]
I agree that FreeOTP+ should show most frequently used tokens in the first page. I am worried about two things
1. Losing muscle memory about the token position because of constant changing positions
2. Overriding the manual orders. It is very confusing to have a dynamic order and manually changed orders.

**Comment by mddvul22** [comment]
I've moved to freeotp+ from andotp.  Dynamic sorting by usage frequency is the one feature of andotp that I really miss and that always makes be ponder whether I should move back to it.  Please implement!

**Comment by helloworld1** [comment]
Would providing a one-time sorting action good for this use case? Feels like always dynamically sorting is conflicting the manual sorting.

**Comment by Minoru** [comment]
There could be a sub-menu "Sorting" where one could choose between "Manual" and "Frequency". The first one is, well, manual, and the second one is dynamic. Maybe there could be other dynamic orderings too, e.g. "Title".
